core/domain/collection_services_test.py

```diff
@@ -1571,12 +1460,12 @@ class CollectionSearchTests(CollectionServicesUnitTests):
     """Test collection search."""
 
     def test_demo_collections_are_added_to_search_index(self):
-        results = collection_services.search_collections('Welcome', 2)[0]
+        results = collection_services.search_collections('Mathematics', 2)[0]
         self.assertEqual(results, [])
 
         collection_services.load_demo('0')
-        results = collection_services.search_collections('Welcome', 2)[0]
-        self.assertEqual(results, ['0'])
+        results = collection_services.search_collections('Mathematics', 2)[0]
+        self.assertEqual(results, [])
```

Comment:
This seems wrong. This test is basically not testing anything. Shouldn't the category still be ``Welcome`` and this line be ``self.assertEqual(results, ['0']);``?